src/content/docs/en/scheme.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "The 'shelter' URI Scheme"
+---
+
+## Introduction
+
+The 'shelter' URI scheme identifies resources that can be reached through the
+Shelter Protocol. These resources are generic and could take various forms.
+For example, a resource could refer to a contract, a manifest or a file part,
+among other options. Each resource is identified by a self-referencing hash,
+which is in the CID format [[CID]](#ref-CID), using the BLAKE2b-256 digest
+[[RFC7693]](#ref-RFC7693) and the Base58 [[BASE58]](#ref-BASE58) encoding.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [[RFC2119]](#ref-RFC2119).
+
+In this document, URIs are enclosed in '<' and '>' as described in Appendix C of
```

Comment:
I would consider putting other things in backticks too like these `<` and `>` symbols